flags.go

```diff
@@ -81,7 +81,7 @@ func (x *cmdLineOpts) parseFlags() error {
 	flag.StringVar(&x.location, "location", defaultLocation, "TestMyNet location")
 	flag.IntVar(&x.datasize, "size", defaultDataSize, "Test size in KBytes")
 	flag.BoolVar(&x.dryrun, "dry-run", false, "Dry-run mode")
-	flag.BoolVar(&x.force, "I-WANT-TO-GET-BANNED", false, "Allow program to hit testmy.net more often than it should.")
```

Comment:
When I wrote this little utility, I contacted testmy.net asking if they'd be interested in listing this as a free tool for their users. They were not that interested, and one of the main reasons is that they were afraid people would unwittingly overload their servers. I promised I'd make it as hard as possible to have it happening. This weirdly named flag makes it clear to people that it should only be considered in extreme cases, whereas "--force" seems like something without much consequence.

Reply:
Thanks for clarification. I did not realize you were working with testmy.net to list this as approved tool. And I also understand why testmy.net is reluctant. I am fine with changing it back, let me know. My only reason to change it to match it like a normal flag. If someone is going to try it often, they are going to do regardless of how it is spelled :)